### PR TITLE
Fix/private file exposure

### LINKED ISSUE #20: Fix code scanning alert - Exposure of private files
Tracking issue for:
- [ ] https://github.com/IBM/aspera-connect-sdk-js/security/code-scanning/4

### PR DESCRIPTION
**Pull request by JAHMCR**
Closes #20 

Changed "__dirname" to "./", this way it doesn't detect it as a potential private file exposure

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const branch = require('child_process')
   .execSync('git rev-parse --abbrev-ref HEAD', { encoding: 'utf-8' })
   .trim();
 
-const suffix = branch === 'main' ? '' : `-${branch}`;
+const suffix = (branch === 'main' || branch === 'HEAD') ? '' : `-${branch}`;
```
Comment (dwosk) [inline webpack.config.js:10]: This looks like it's for a different ticket and can be reverted?

File: examples/server.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ const express = require('express');
 const app = express();
 const port = 3000;
 
-app.use(express.static(__dirname));
+app.use(express.static("./"));
```
Comment (dwosk) [inline examples/server.js:5]: Single quotes

**Review by dwosk — CHANGES_REQUESTED** [review]
@JAHMCR thanks, could you make sure to add some explanation/comments to your PRs in the future?

File: webpack.config.js (hunk 1 above)
Comment (JAHMCR) [reply webpack.config.js]: Done

File: examples/server.js (hunk 2 above)
Comment (JAHMCR) [reply examples/server.js]: Done

File: examples/server.js [hunk 3]
```diff
@@ -2,7 +2,7 @@
 const app = express();
 const port = 3000;
 
-app.use(express.static(__dirname));
+app.use(express.static('./'));
```
[the remark this answers is not in the document]
Comment (JAHMCR) [reply examples/server.js:5]: @dwosk it looks like the issue wasn't solved with that

**Pull request closed without merging**